demisto_sdk/commands/content_graph/neo4j_service.py

```diff
@@ -101,31 +130,12 @@ def start():
     _download_apoc()
     docker_client = init_global_docker_client()
     _stop_neo4j_service_docker(docker_client)
-    docker_client.containers.run(
-        image=NEO4J_SERVICE_IMAGE,
-        name="neo4j-content",
-        ports={"7474/tcp": 7474, "7687/tcp": 7687, "7473/tcp": 7473},
-        volumes=[
-            f"{REPO_PATH / NEO4J_FOLDER / NEO4J_DATA_FOLDER}:/{NEO4J_DATA_FOLDER}",
-            f"{REPO_PATH / NEO4J_FOLDER / NEO4J_IMPORT_FOLDER}:{LOCAL_NEO4J_PATH / NEO4J_IMPORT_FOLDER}",
-            f"{REPO_PATH / NEO4J_FOLDER / NEO4J_PLUGINS_FOLDER}:/{NEO4J_PLUGINS_FOLDER}",
-        ],
-        detach=True,
-        environment={
-            "NEO4J_AUTH": f"neo4j/{NEO4J_PASSWORD}",
-            "NEO4J_apoc_export_file_enabled": "true",
-            "NEO4J_apoc_import_file_enabled": "true",
-            "NEO4J_apoc_import_file_use__neo4j__config": "true",
-            "NEO4J_dbms_security_procedures_unrestricted": "apoc.*",
-            "NEO4J_dbms_security_procedures_allowlist": "apoc.*",
-        },
-        healthcheck={
-            "test": f"curl --fail {NEO4J_DATABASE_HTTP} || exit 1",
-            "interval": 5 * 1000000000,
-            "timeout": 15 * 1000000000,
-            "retries": 10,
-        },
-    )
+    try:
+        _docker_start()
+    except Exception as e:
+        logger.debug(f"Could not start neo4j container, deleting data folder and trying again. {e}")
```

Comment:
```suggestion
        logger.debug(f"Could not start neo4j container, delete data folder and trying again. {e}")
```
are you sure this is the only possible error?

Reply:
This is not the only possible error, but it's possible one that I had when run locally in the new version.
In case of another error, `_docker_start` will fail again.